.gitignore

```diff
@@ -1,2 +1,19 @@
-_build
+# backup files
+*~
+*.bak
+.DS_Store
+
+# project file
 *.hdp
+
+# documentation build directory
+build/
```

Comment:
I think it would be better to use `/documentation/build/` here. (You could also remove the comment then.)

Reply:
If the pattern doesn’t start with a slash, it matches files and directories in any directory or subdirectory. I can copy and paste the same .gitignore file with different directory names like 'docs' or 'documentation'. Do you think is better to be more specific in each project?

Reply:
I like to be specific. I think accidentally hiding a valid directory is worse than showing it in the `git status` output.  (In fact I use the Git alias `s = status --ignored` so that I always see the ignored files.)

If you prefer it this way I don't insist.